### DISCUSSION TITLE
Failure to install the package of roommate

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#237] [Archive]
@richardreeve @soniamitchell Good morning all. Sorry, I had a problem while installing the roommate package and gave me this error message:
Using github PAT from envvar GITHUB_PAT
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 404.
  Not Found

  Did you spell the repo owner (`IBAHCM`) and repo name (`AwenaSangsterSeries03`) correctly?
  - If spelling is correct, check that you have the required permissions to access the repo.

I actually spelled the names of repo owner and repo name correctly. Why this happens?
Thanks.

**Comment by richardreeve** [comment]
Apologies for the delay, but this is our standard problem that some people have installing private packages. The solution seems to be to clone the repo and then use `devtools::install()`.

**Reply by soniamitchell** [reply]
@richardreeve Bakari's error might be a bug in remotes / devtools https://github.com/r-lib/remotes/issues/321 
@Nambunga's error might be related. Hopefully they fix it for next year's class!
